### PR TITLE
Internationalization issue  [login controller]

### PR DESCRIPTION
**Pull request by KasunApplications**
First steps for issues #203

Screenshots for the change: 

screen shot 2017-08-15 at 11 44 24 pm

### CODE REVIEW

File: Susi.xcworkspace/xcshareddata/Susi.xcscmblueprint [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositorySystemKey" : "com.apple.dt.Xcode.sourcecontrol.Git",
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryIdentifierKey" : "3176722731A10ADB3E49DAAD26D6F943DA28A0A9"
+    },
+    {
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryURLKey" : "https:\/\/github.com\/marmelroy\/Localize-Swift.git",
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositorySystemKey" : "com.apple.dt.Xcode.sourcecontrol.Git",
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryIdentifierKey" : "777BAFA1E8F9825A0B505DD4BE1D79238ECB3DCB"
+    }
+  ]
+}
```
Comment (sudheesh001) [inline Susi.xcworkspace/xcshareddata/Susi.xcscmblueprint]: New line

File: Podfile [hunk 2]
```diff
@@ -23,6 +23,7 @@ target 'Susi' do
   pod 'NVActivityIndicatorView'
   pod 'Fakery'
   pod 'M13Checkbox', :git => 'https://github.com/Marxon13/M13Checkbox.git'
+  pod 'Localize-Swift'
```
Comment (sudheesh001) [inline Podfile]: 👍

File: Susi.xcworkspace/xcshareddata/Susi.xcscmblueprint (hunk 1 above)
Comment (KasunApplications) [inline Susi.xcworkspace/xcshareddata/Susi.xcscmblueprint]: pls review

**Comment by chashmeetsingh** [comment]
@mariobehling can you please review the translations [here](https://github.com/KasunApplications/susi_iOS/blob/6ef9db8f64aa1848e0be2ef0412209904b51d07f/Susi/Helpers/de.lproj/Localizable.strings).
Thanks!

**Comment by isuruAb** [comment]
@sudheesh001  can you review this again . he has done documentation part in another issue #210

**Comment by sudheesh001** [comment]
Please address the codacy issues

**Comment by isuruAb** [comment]
@KasunApplications  please resolve conflicts

**Pull request closed without merging**